src/Sulu/Bundle/ContactBundle/Controller/ContactController.php

```diff
@@ -679,6 +675,25 @@ public function getSecurityContext()
         return 'sulu.contact.people';
     }
 
+    /**
+     * Takes an array with contact-data and adds the corresponding avatar-data.
+     *
+     * @param array $contacts
+     * @param String $locale
+     *
+     * @return array of contacts-data with added avatar-data
+     */
+    private function addAvatars($contacts, $locale)
+    {
+        foreach ($contacts as $key => $contact) {
+            $contacts[$key]['avatar'] = $this->getContactManager()
+                ->getById($contact['id'], $locale)
+                ->getAvatar()['thumbnails'];
+        }
```

Comment:
This results in another database request for every single contact, you really should solve that in a different way.